docs/conf.py

```diff
@@ -117,6 +119,20 @@
 # The full version, including alpha/beta/rc tags.
 release = __version__
 
+# Replace $variables in the .rst files if on a readthedocs worker
+if "/home/docs/checkouts" in " ".join(sys.argv):
+    mss_search = subprocess.run(["conda", "search", "-c", "conda-forge", "mss"], stdout=subprocess.PIPE,
+                                stderr=subprocess.STDOUT, encoding="utf8").stdout
+    mss_search = "   ".join([line for line in mss_search.splitlines(True) if line.startswith("mss ")][-2:])
+
+    for file in os.listdir():
```

Comment:
just a comment someone else falls over this
file was in Py 2 a builtin and this was removed in Py 3 by the io framework